tests/sanity/ignore-2.10.txt

```diff
@@ -1104,6 +1104,8 @@ plugins/modules/net_tools/ldap/ldap_entry.py validate-modules:doc-missing-type
 plugins/modules/net_tools/ldap/ldap_entry.py validate-modules:parameter-type-not-in-doc
 plugins/modules/net_tools/ldap/ldap_passwd.py validate-modules:doc-missing-type
 plugins/modules/net_tools/ldap/ldap_passwd.py validate-modules:doc-required-mismatch
+plugins/modules/net_tools/ldap/ldap_search.py validate-modules:doc-missing-type
```

Comment:
Generally speaking, we shouldn't add new entries to `ignore.txt`

Could you please remove these two lines and update the `DOCUMENTATION`. The test output should say what needs fixing.